docker-compose.yml

```diff
@@ -0,0 +1,43 @@
+version: '3'
+services:
+  poet-mongo:
+    image: mongo:3.4
+    ports:
+      - "27017:27017"
+  poet-rabbit:
+    image: rabbitmq:3
+    ports:
+      - "5671:5671"
+      - "5672:5672"
+      - "15672:15672"
+    hostname: my-rabbit
+  poet-ipfs:
+    image: jbenet/go-ipfs:latest
+    ports:
+      - "8080:8080"
+      - "4001:4001"
+      - "5001:5001"
+    volumes:
+      - .ipfs-docker:/data/ipfs
+  poet-node:
+    build: .
+    ports:
+      - "18080:18080"
```

Comment:
Are these ports correct? I think I saw integration tests are looking for `:1080`? @wzalazar @lautarodragan

Reply:
poet-node runs on 18080 by default, and consumes mongodb, ipfs and rabbitmq in their default ports.